.github/workflows/nightly-tests.yml

```diff
@@ -30,13 +30,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: anybody/setup-anybody@main
-        name: Install AnyBody
-        with:
-          anybody-version: "7.5.0"
-          anybody-version-suffix: "Beta"
-
-      - uses: prefix-dev/setup-pixi@v0.4.3
+      - uses: prefix-dev/setup-pixi@v0.5.0
         with:
           cache: true
```

Comment:
you need `environments: test` here. Otherwise it will only cache the `default` environment.